pkg/machine/shim/host.go

```diff
@@ -283,6 +283,7 @@ func checkExclusiveActiveVM(provider vmconfigs.VMProvider, mc *vmconfigs.Machine
 			return err
 		}
 		if state == machineDefine.Running || state == machineDefine.Starting {
+			fmt.Printf("Cannot start \"%s\" if machine \"%s\" is running:\n\n\tpodman machine stop %s\n\tpodman machine start %s\n\n", mc.Name, name, name, mc.Name)
```

Comment:
only reason why I preferred this over detailed err message is because we already hint to start the machine when we just do `init` so this is just another minor troubleshooting. but I also realize that for `init` is just hinting the next step while here it's suggesting a fix, and that's quite different.

Reply:
maybe the error should say something like this?
```
unable to start "vmY": machine "vmX" is already running or starting and there can only be one running machine at a time
```

Reply:
oh actually, we have an error for that which is `ErrMultipleActiveVM` just noticed now, is it a breaking change to change the error? I believe it makes more sense to use that

Reply:
Error messages are not stable so we can change them, for the error types you should check if callers do some errors.Is()/As() checks on them, if so you need to update the callers but there are not stable either as we do not offer stable APIs except for packages in pkg/bindings.

Reply:
thanks a lot. I'll check and update the PR